src/data.js

```diff
@@ -0,0 +1,65 @@
+let todos = 0;
+const defaultTodo = [
+  {
+    description: 'Eat',
+    completed: false,
+    id: 0,
+    index: 0,
+  },
+  {
+    description: 'Sleep',
+    completed: false,
+    id: 1,
+    index: 1,
+  },
+  {
+    description: 'Code',
+    completed: false,
+    id: 2,
+    index: 2,
+  },
+  {
+    description: 'Repeat',
+    completed: false,
+    id: 2,
+    index: 3,
+  },
+];
```

Comment:
- [ ] I think since we are not using these default tasks, we don't actually need to keep them inside this array, this will help keep our code clean 😉